app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -1,23 +1,35 @@
 import React, { useState } from "react";
 
 import { Formik, Form } from "formik";
+import { Check } from "neetoicons";
 import { Button, Pane } from "neetoui";
-import { Input, Textarea } from "neetoui/formik";
+import { Input, Select } from "neetoui/formik";
 
-import notesApi from "apis/notes";
+import { CONTACTS, TAGS } from "components/Dashboard/constants";
+import { buildSelectOption } from "utils/index";
 
 import { NOTES_FORM_VALIDATION_SCHEMA } from "../constants";
 
-const NoteForm = ({ onClose, note, isEdit }) => {
+const tags = TAGS.map(buildSelectOption);
+const contacts = CONTACTS.map(contact =>
+  buildSelectOption(`${contact.firstName} ${contact.lastName}`)
+);
+
+const parseValues = values => {
+  const assignedContact = values.assignedContact.label;
+  const tags = values.tags.map(tag => tag.label);
+
+  return { ...values, tags, assignedContact };
+};
+
+const NoteForm = ({ onClose, note, isEdit, handleNote }) => {
```

Comment:
Go through the Slack thread where I have even put up a video regarding unnecessary namespacing.

Reply:
Same as before, `NoteForm` was part of the base [`wheel code`](https://github.com/bigbinary/wheel/blob/fb44f4f583d0fa4de9d69f853c33c4f963a1aa38/app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx#L11), hence it went unnoticed.

But I'll make sure to check the `base` code as well in the upcoming commits.

Reply:
@vivek-viswam-rv Go through https://academy.bigbinary.com/learn-react/react-best-practices/miscellaneous-tips#:~:text=Importing%20under%20an%20alias

Here import it as `FormikForm`.